doc/source/tutorial/stats/sampling.rst

```diff
@@ -135,16 +148,16 @@ distribution by calling the ``rvs`` method:
 
 .. note:: Please note the difference between the `rvs` method of the
           distributions present in :mod:`scipy.stats` and the one provided
-          by these generators. In general, the aim of the generation
-          methods implemented in :class:`~rv_continuous` is to provide
-          generators that work reasonably well for common use cases, e.g.
-          to generate small samples using different shape parameters or to
-          generate large samples for a fixed shape parameter. Depending on
-          the goal, specialized methods can lead to a substantial speedup.
-          For example, at the cost of a rather expensive setup step, very
-          fast generation can be achieved for large samples and a fixed
-          shape parameter. Also, even if the same URNG (``seed``) is used,
-          the resulting rvs will be different in general.
+          by these generators. Even if the same URNG (``seed``) is used,
+          the resulting rvs will be different in general: the implementation
+          of `rvs` in :class:`~rv_continuous` usually relies on the NumPy
+          module `random` for well-known distributions (e.g., for the normal
+          distribution, the beta distribution) and transformations of other
+          distributions (e.g., normal inverse Gaussian `norminvgauss` and the
+          lognormal distribution). If no specific  method is implemented,
+          `rv_continuous` defaults to a numerical inversion method of the CDF
+          that is very slow. As described in this tutorial, UNU.RAN implements
+          various approaches that will generally produce different samples.
```

Comment:
This elucidates some of the subtle concepts in the tutorial. I went through the diff and it looks good to me. Thanks for working on this, @chrisb83! I will add the table next Monday.

How does the new description sound to you, @tupui? (It's based on https://github.com/scipy/scipy/pull/14215#discussion_r672824528)

Reply:
Sound better for me as well, thanks! Just a few suggestions for rendering.